### PR TITLE
Adding JedisSentinelPool support in RediSearch  Client

### PR DESCRIPTION
**Pull request by chandu-atina**
Hello @itamarhaber @mnunberg @dvirsky 
Implemented JedisSentinelPool support in RediSearch Client. Could you guys please review the PR and provide your feedback? I will add the JUnit test cases for the changes based on your feedback.

### CODE REVIEW

**Comment by itamarhaber** [comment]
Thanks @chandu-atina 

👍 - @gkorland?

**Comment by chandu-atina** [comment]
@itamarhaber
There are 3 pull requests in open state from me. Could anyone of the maintainers review them and provide the feedback. 

I would really appreciate if you guys can spend some time and review the PRs

**Comment by chandu-atina** [comment]
@itamarhaber @gkorland
Did you guys get a chance to look into the PR

**Comment by chandu-atina** [comment]
@gkorland I have updated the PR, could you please review it

File: src/main/java/io/redisearch/client/ClusterClient.java [hunk 1]
```diff
@@ -26,6 +28,59 @@ public ClusterClient(String indexName, String host, int port) {
...
+     * <ul><li> password - NULL, no authentication required to connect to Redis Server</li></ul>
+     * 
+     * @param indexName the name of the index we are connecting to or creating
+     * @param masterName the masterName to connect from list of masters monitored by sentinels
+     * @param sentinels the set of sentinels monitoring the cluster
+     * @param timeout the timeout in milliseconds
+     * @param poolSize the poolSize of JedisSentinelPool
+     */
+    public ClusterClient(String indexName, String masterName, Set<String> sentinels, int timeout, int poolSize) {
+        super(indexName, masterName, sentinels, timeout, poolSize, null);
```
[the remark this answers is not in the document]
Comment (chandu-atina) [reply src/main/java/io/redisearch/client/ClusterClient.java]: ahhh, my bad. Changed!!

**Comment by chandu-atina** [comment]
@gkorland I have amended the commit to include the changes as suggested

**Pull request merged**